playbooks/roles/ostree/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+katello_devel: False
```

Comment:
It's usually convention to prefix these role variables with the name of the role